fairroot/base/sim/FairVolumeList.h

```diff
@@ -30,7 +30,7 @@ class FairVolumeList : public TObject
     TObjArray fData;
 
   public:
-    FairVolumeList();
+    FairVolumeList() { fData.SetOwner(kTRUE); }
```

Comment:
Consider squashing this part into "feat(Base): Avoid extra allocation in `FairVolumeList`"?

Reply:
Ya, good idea, will do on the next edit round

Reply:
done